axelor-account/src/main/resources/domains/AccountChart.xml

```diff
@@ -11,6 +11,8 @@
     <string name="countryCode" title="Country code"/>
     <many-to-one name="metaFile" ref="com.axelor.meta.db.MetaFile" title="File"/>
     <many-to-many name="companySet" ref="com.axelor.apps.base.db.Company"/>
+    <integer name="accountCodeMinNbrChar" title="Account code min. nbr. char"
+      selection="account.code.nbr.char.select" required="true"/>
```

Comment:
<html>
<body>
<!--StartFragment--><google-sheets-html-origin><!--td {border: 1px solid #ccc;}br {mso-data-placement:same-cell;}-->

Select name |   | Example
-- | -- | --
module.name.object.name.field.name.select |   | sale.sale.order.sale.order.status.select
  |   |  
For the module name, don't write "axelor", for example if the module name is "axelor-sale", it will be just "sale". |   |  

<!--EndFragment-->
</body>
</html>